### PR TITLE
Rename method arguments to use camelCase casing (plugin directory)

### PR DESCRIPTION
**Pull request by SharkyKZ**
Partial Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30988.

### Summary of Changes

Renames some method arguments to use camelCase casing.

### Testing Instructions

Thorough code review / Joomla works like before.

### Documentation Changes Required

No.

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c87c8877e09e37d0f37c9a85279d9e43dcd0f217<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31151">issues.joomla.org/tracker/joomla-cms/31151</a>.</sub>

File: plugins/content/contact/contact.php [hunk 1]
```diff
@@ -93,25 +93,25 @@ public function onContentPrepare($context, &$row, $params, $page = 0)
 	/**
 	 * Retrieve Contact
 	 *
-	 * @param   int  $created_by  Id of the user who created the contact
+	 * @param   int  $userId  Id of the user who created the article
```
Comment (zero-24) [inline plugins/content/contact/contact.php:96]:
```suggestion
	 * @param   int  $userId  Id of the user who you want to get the contact data
```

Comment (SharkyKZ) [reply plugins/content/contact/contact.php:96]: Is this correct English though?

Comment (zero-24) [reply plugins/content/contact/contact.php:96]: I'm sure it is not but it gets closer to what it happening here right?

Comment (infograf768) [reply plugins/content/contact/contact.php:96]: what about 
`* @param   int  $userId  Id of the user from whom to get the contact data`

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on c87c8877e09e37d0f37c9a85279d9e43dcd0f217<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31151">issues.joomla.org/tracker/joomla-cms/31151</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31151">issues.joomla.org/tracker/joomla-cms/31151</a>.</sub>

**Pull request merged**